### PR TITLE
[Item Mods] Update all Double/Triple Attack damage gear

### PR DESCRIPTION
**Pull request by Schaedelschaden**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Updates all equipment that uses the DOUBLE_ATTACK_DMG and TRIPLE_ATTACK_DMG mods created in #2080

Affects the following gear:
**Double Attack damage** - Widowmaker, Warrior’s Cuisses +2, Devivifier, Agoge Cuisses, Agoge Cuisses +1, Agoge Cuisses +2, Agoge Cuisses +3, Cichol’s Mantle, Quiahuiz Helm, Quiahuiz Trousers
**Triple Attack damage** - Demonry Ring, Triplus Dagger, Assassin's Poulaines +2, Chiner's Belt, Chiner's Belt +1, Plunderer's Poulaines, Plunderer's Poulaines +1, Plunderer's Poulaines +2, Plunderer's Poulaines +3, Assassin's Gorget, Assassin's Gorget +1,  Assassin's Gorget +2, Hetairoi Ring, Toutatis's Cape

## Steps to test these changes

Update the local database with the item_mod.sql
Add the equipment to your character
Whack some mobs and see if your double/triple attack white damage increases from your normal white damage

Useful commands:
!setmod 1038 1-100 charName (Sets DOUBLE_ATTACK_DMG to specified value)
!setmod 1039 1-100 charName (Sets TRIPLE_ATTACK_DMG to specified value)
!setmod 288 1-100 charName (sets DOUBLE_ATTACK proc rate to specified value)
!setmod 302 1-100 charName (sets TRIPLE_ATTACK proc rate to specified value)

### CODE REVIEW

**Comment by Schaedelschaden** [comment]
Ignore the weaponskills.lua and attackround.cpp files, those were not supposed to commit.

**Comment by Schaedelschaden** [comment]
Found Cichol's Mantle missing DEF mod
Found Plunderer's Poulaines +2 missing all stats

**Comment by Schaedelschaden** [comment]
I am hands off on commits for this PR.

PR is ready for review.

**Comment by TeoTwawki** [comment]
WHile here, can you rename these 
```
    DA_DOUBLE_DAMAGE         = 408, // Double attack's double damage chance %.
    TA_TRIPLE_DAMAGE         = 409, // Triple attack's triple damage chance %.
```
to
```
    DA_DOUBLE_DMG_RATE         = 408, // Double attack's double damage chance %.
    TA_TRIPLE_DMG_RATE         = 409, // Triple attack's triple damage chance %.
```
so that people don't use them incorrectly again? They resist in `/scripts/globals/status.lua` and `/src/map/modifier.h`

**Comment by TeoTwawki** [comment]
> I am hands off on commits for this PR.
> 
> PR is ready for review.

well, I timed my post poorly, haha >.> ha >.>

**Comment by Schaedelschaden** [comment]
Let me look into where those are used real quick so I don't break them by changing their names.

**Comment by Schaedelschaden** [comment]
It would probably be better to throw those into a new PR to keep them separate.

File: sql/item_mods.sql [hunk 1]
```diff
@@ -62485,6 +62551,7 @@ INSERT INTO `item_mods` VALUES (27153,31,80);   -- MEVA: 80
 INSERT INTO `item_mods` VALUES (27153,68,27);   -- EVA: 27
 INSERT INTO `item_mods` VALUES (27153,288,4);   -- DOUBLE_ATTACK: 4
 INSERT INTO `item_mods` VALUES (27153,384,600); -- HASTE_GEAR: 600
+INSERT INTO `item_mods` VALUES (27152,1038,7);  -- DOUBLE_ATTACK_DMG: 7
```
Comment (Xaver-DaRed) [inline sql/item_mods.sql]: wrong item id

Comment (Schaedelschaden) [reply sql/item_mods.sql]: Corrected

**Comment by TeoTwawki** [comment]
looks like you accidently overwrote your changes in that force push. We can get this re-open still by pushing them back to same branch as long as you do not delete it

**Comment by TeoTwawki** [comment]
if intending to make your "master" branch 1:1 with it, I suggest waiting till after this merges or making a new "main" and setting that as your default instead of using "master"

**Comment by Schaedelschaden** [comment]
Hmm, I was trying to get my master branch reset to base LSB so I could change the pull request branch. Had tried following the info from https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request and seems like I messed it up. /headdesk

**Comment by TeoTwawki** [comment]
not sure if you will need it, but this may be of interest to you for finding your previous commit before this happened https://www.w3docs.com/learn-git/git-reflog.html

if you get your changes back in here, github will light the "reopen pull request" button up and we can resume

**Comment by Schaedelschaden** [comment]
I really messed up the master branch and wasn't able to figure out how to get the proper commit restored. I started a completely brand new repository based on LSB's most recent version, created a new branch, and changed only the item_mod.sql file. The new PR is #2105 and should be available for review.

**Pull request closed without merging**